submissions/usides/js-dom/dist/css/main.css

```
@@ -0,0 +1,449 @@
+@import url("https://fonts.googleapis.com/css2?family=Bai+Jamjuree:wght@700&display=swap");
+.container {
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+button,
+a,
+input,
+li {
+  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+}
+
+li {
+  list-style-type: none;
+}
+
+html,
+body {
+  height: 100%;
+}
+
+body {
+  background-color: #003154;
+  font-family: "Bai Jamjuree", sans-serif;
+  text-transform: uppercase;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+      -ms-flex-direction: column;
+          flex-direction: column;
+  color: #fff;
+}
+
+header {
+  position: fixed;
+  width: 100%;
+  background-color: #001c31;
+  z-index: 1;
+}
+
+header .container {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  border-left: 3px solid #2d5c7d;
+  border-right: 3px solid #2d5c7d;
+}
+
+.header-left-gr {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  margin-left: 1.5rem;
+}
+
+.header-left-gr img {
+  width: 3.5rem;
+  padding: 0.8rem 0;
+}
+
+.header-left-gr h1 {
+  font-size: 1.8rem;
+  line-height: 1.1;
+  margin-left: 1rem;
+}
+
+.next-btn {
+  border: 2px solid #fff;
+  padding: 0.7rem 1rem;
+  margin-right: 1.5rem;
+  -webkit-transition: background-color 300ms ease;
+  transition: background-color 300ms ease;
+}
+
+.next-btn:active {
+  background-color: #930000;
+}
+
+.menu-btn {
+  display: none;
+}
+
+.center-content {
+  margin-top: 6rem;
+  -webkit-box-flex: 1;
+      -ms-flex: 1 0 auto;
+          flex: 1 0 auto;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  width: 100%;
+}
+
+nav.side-nav {
+  -ms-flex-preferred-size: 230px;
+      flex-basis: 230px;
+  background-color: #003154;
+  border-left: 3px solid #2d5c7d;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+      -ms-flex-direction: column;
+          flex-direction: column;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+}
+
+nav.side-nav img {
+  -ms-flex-item-align: center;
+      -ms-grid-row-align: center;
+      align-self: center;
+  max-width: 9rem;
+  margin-bottom: 2rem;
+}
+
+ul.side-nav-menu .menu-link {
```

Avoid using such heavy selectors, please.
`.menu-link` would be enough here
```suggestion
.menu-link {
```

---

Done